roles/addons/clustershell/tasks/main.yml

```diff
@@ -1,9 +1,9 @@
-- name: Package
+- name: package █ Install clustershell package
   package:
     name: clustershell
     state: present
 
-- name: Template >> /etc/clustershell/groups.d/local.cfg
+- name: template █ Generate /etc/clustershell/groups.d/local.cfg
   template:
     src: local.cfg.j2
     dest: /etc/clustershell/groups.d/local.cfg
```

Comment:
missing template tag

Reply:
Can you double check ? I see it in my version.

Reply:
Definitely here.